Test error under windows strawberry perl 5.28

```
t/safe.t ............... ok
t/safe2.t .............. ok
t/safe3.t .............. ok
t/strict.t ............. ok
t/taint.t .............. 1/21 Error in tempfile() using template \XXXXXXXXXX: Could not create temp file \cFxqKlepwS: Permission denied at t/taint.t line 12.
# Looks like your test exited with 13 just after 1.
t/taint.t .............. Dubious, test returned 13 (wstat 3328, 0xd00)
Failed 20/21 subtests
t/template-encoding.t .. ok
t/warnings.t ........... ok

Test Summary Report
-------------------
t/taint.t            (Wstat: 3328 Tests: 1 Failed: 0)
  Non-zero exit status: 13
  Parse errors: Bad plan.  You planned 21 tests but ran 1.
Files=22, Tests=164,  8 wallclock secs ( 0.12 usr +  0.08 sys =  0.20 CPU)
Result: FAIL
Failed 1/22 test programs. 0/164 subtests failed.
gmake: *** [Makefile:880: test_dynamic] Error 255
FAIL
! Installing M/MS/MSCHOUT/Text-Template-1.53.tar.gz failed. See C:\Users\zdm\AppData\Local\Temp\.cpanm\work\1533377208.15244\build.log for details. Retry with --force to force install it.
```

--- a/t/taint.t
+++ b/t/taint.t
@@ -9,7 +9,7 @@ use File::Temp;
 
 use_ok 'Text::Template' or exit 1;
 
-my $tmpfile = File::Temp->new;
+my $tmpfile = File::Temp->new( $^O eq 'MSWin32' ? (DIR => '.') : () );
 my $file    = $tmpfile->filename;
 
 # makes its arguments tainted

